lib/packer.js

```diff
@@ -15,9 +17,11 @@ var Packer = module.exports = function(options) {
   this._options = options;
 
   options.deflateChunkSize = options.deflateChunkSize || 32 * 1024;
-  options.deflateLevel = options.deflateLevel != null ? options.deflateLevel : 9;
-  options.deflateStrategy = options.deflateStrategy != null ? options.deflateStrategy : 3;
+  options.deflateLevel = options.deflateLevel !== null ? options.deflateLevel : 9;
```

Comment:
took me a while to find this - its why some tests don't pass.. it should be `!= null` so that undefined goes to the default...
I've fixed it in my branch